pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2020-07-08T06:00:00+01:00
+type: Communication
+tags:
+  - Git
+  - GitHub
+  - VS Code
+  - Markdown
+cover: img/mistake odyssey-Carolinie-cavalli.webp
+authors:
+  - name: Barbaño González
+    photo: /img/avatars/barbano.webp
+    link: https://www.linkedin.com/feed/
```

Comment:
This link is not right. I think this is intended to point at the user profile.

Reply:
Thanks 👍